### PR TITLE
Turn on and off TTL switch of a specific DDS

### LINKED ISSUE #82: Turn on and off TTL switch of a specific DDS
### Feature you want to implement
Similar to #79, I will implement to turn on and off the TTL switch, which controls the DDS output.

#### URL format
`~/dds/switch/?device={device}&channel={channel}&on={on}`

### PR DESCRIPTION
**Pull request by BECATRUE**
This closes #82.

For a request, refer to the following command.
```shell
$ curl -X POST 'http://127.0.0.1:8000/dds/switch/?device=urukul0&channel=1&on=True'
```

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -634,6 +634,49 @@ def run(self):
     return rid
 
 
+@app.post("/dds/switch/")
+async def set_dds_switch(device: str, channel: int, on: bool):
```
Comment (kangz12345) [inline main.py]: Please annotate the return type.

**Review by kangz12345 — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (kangz12345) [reply main.py]: Also please check the reviews in #84 as it has a similar structure.

**Review by kangz12345 — CHANGES_REQUESTED** [review]
Please push your work!

**Comment by BECATRUE** [comment]
Oh, I now push my work! Thanks to notify me😃

**Review by kangz12345 — APPROVED** [review]
LGTM

**Pull request merged**